### PR TITLE
Compile UP protobuf definitions as feature

### PR DESCRIPTION
**Pull request by franklinselva**
The UP protobuf definitions can now be compiled at request as a `feature` through the following:

```bash
cargo build --features=unified_planning
```

### CODE REVIEW

**Comment by arbimo** [comment]
That's an excellent idea!

To complete it, you should:
 - make the `tonic-build` dependency only active when the feature is enabled.
 - document this in a `grpc/api/README.md` file

**Comment by franklinselva** [comment]
@arbimo  I have updated the dependencies and documentation for building the definitions. 

As a side note, this change allows the `up-server` to be cross compiled for windows platform.

File: grpc/README.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 This module holds the functionality for the gRPC library integrated into Aries. The implementation is currently focused towards [**unified_planning**](https://github.com/aiplan4eu/unified-planning).
 
+<!-- TODO: Update README for UP Server and Usage Information -->
+
+## GRPC API
```
Comment (arbimo) [inline grpc/README.md:7]: Just need to be a little more specific: generating the rust bindings (file XXXX) from the protobuf format definition (file XXX).

**Review by arbimo — CHANGES_REQUESTED** [review]
Since it is not done systematically, the CI should also be updated to check that the unified_planning.rs file is up to date. 
This should be as simple as compiling the `grpc/api` crate with the feature enabled.

Finally the feature should have a more specific name, e.g., `generate_bindings`. (the whole crate is about unified planning).

**Comment by franklinselva** [comment]
The current bindings are slightly custom (with comments and formatting) and launching the CI might be failing at this stage. Shall we update the current bindings to the generated version?

The current differences when compiling bindings are available in the patch ([bindings.txt](https://github.com/plaans/aries/files/10021050/bindings.txt)).

**Review by arbimo — APPROVED** [review]

**Pull request merged**